### PR TITLE
chore: add GitHub repository integration as prerequisite

### PR DESCRIPTION
**Pull request by gnumoksha**
https://github.com/getsentry/action-release/issues/12#issuecomment-663879593

### CODE REVIEW

**Comment by ceorourke** [comment]
Tagging in @mgaeta who is more familiar with the CI on this repo than I am - any idea why the test is failing?

File: README.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ Give your new integration a name (for example, "GitHub Action Release Integratio
 
 Click “Save” at the bottom of the page and grab your token, which you’ll use as your `SENTRY_AUTH_TOKEN`. We recommend you store this as an [encrypted secret](https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets).
 
+#### Create a GitHub repository integration
```
Comment (mgaeta) [inline README.md:22]:
```
#### Install GitHub integration
```

File: README.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ Give your new integration a name (for example, "GitHub Action Release Integratio
...
+For this action to find your commits, your GitHub repository needs to be added. In Sentry, navigate to _Settings > Integrations > GitHub_. Go to the "Configurations" tab and click "Configure". Confirm that your repository is there, if not click "Add Repository".
```
Comment (mgaeta) [inline README.md:23]:
```
A GitHub repository must be added to Sentry in order for this action to find your commits.
```

**Review by mgaeta — CHANGES_REQUESTED** [review]
I have some minor copy changes. The test failures look like a problem on our end. I'll start investigating that now.

**Pull request closed without merging**